index.html

```diff
@@ -1,16 +1,106 @@
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
+    <header>
+      <div class="wrapper-header">
+        <div class="logo"></div>
+        <div class="nav-bar">
+          <img
+            src="/images/hamburgericon.png"
+            alt="hamburger menu icon"
+            class="hamburger-menu"
+          />
+
+          <h1 class="page-header">
+            The best cakes in town delivered to your door
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav>
+      <ul class="navigation">
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Cakes</a></li>
+        <li><a href="#">Orderin</a></li>
+        <li><a href="#">Lessons</a></li>
+        <li><a href="#">About</a></li>
+      </ul>
+    </nav>
+
+    <main>
+      <div class="container">
+        <section class="welcome">
+          <div class="wrapper-welcome">
+            <h2>Welcome</h2>
+            <p>
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Rem ea
+              molestiae temporibus ipsa eaque repellat omnis sunt esse?
+              Repellendus consequuntur iusto repudiandae, maiores sint quos
+              accusantium. Sunt facere doloribus obcaecati culpa libero, nam,
+              aperiam, optio assumenda ex et autem amet. Consectetur
+              voluptatibus natus aliquam harum distinctio incidunt accusamus
+              fugiat quos.
+            </p>
+          </div>
+        </section>
+
+        <section class="cake">
+          <div class="wrapper-cake"></div>
+        </section>
+      </div>
+
+      <section class="cake-images">
+        <div class="wrapper-images">
+          <div class="img-container 1">
+            <img
+              src="/images/cake images/alexandra-gornago-_B7shfNUXEA-unsplash.jpg"
+              alt="cake"
+              id="cake1"
+            />
+          </div>
+          <div class="img-container 2">
+            <img
+              src="/images/cake images/anthony-espinosa-6iqpLKqeaE0-unsplash.jpg"
+              alt="cake"
+              id="cake2"
+            />
+          </div>
+          <div class="img-container 3">
+            <img
+              src="/images/cake images/deva-williamson-tW0Ix_Ajg6Y-unsplash.jpg"
+              alt="cake"
+              id="cake3"
+            />
+          </div>
+          <div class="img-container 4">
+            <img
+              src="/images/cake images/kaouther-djouada-hcEDfkiVmMI-unsplash.jpg"
+              alt="cake"
+              id="cake4"
+            />
+          </div>
+        </div>
+      </section>
+    </main>
 
-</html>
\ No newline at end of file
+    <footer>
+      <div class="contact" id="contact">
+        <p>Get in touch with us to experience the most delicious cakes</p>
+        <p>Email:<strong> forexample@cakes.com</strong></p>
```

Comment:
Sometimes a cleaner way to put spaces in with html is to use a non-breaking space `&nbsp;`, so in this case it would look like `<p>Email:&nbsp;<strong>forexample@cakes.com</strong></p>`

Reply:
Thanks Matthew. I didn't know about the "&nbsp" in html. Will use it from now on